Annotations/TestHelper.Monkey.Annotations.asmdef

```diff
@@ -1,7 +1,8 @@
 {
     "name": "TestHelper.Monkey.Annotations",
-    "rootNamespace": "TestHelper.Monkey",
+    "rootNamespace": "TestHelper.Monkey.Annotations",
```

Comment:
Set `rootNamespace` to the root namespace of the package.

See: https://www.nowsprinting.com/entry/2023/02/17/004304

Reply:
Fixed at a1018e7.